temporal/t.downgrade/t.downgrade.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
```

Comment:
```suggestion
```

Reply:
There must be a reason why many (all?) other `t.*` modules have this line. If this line is indeed not needed, it should be removed from all python scripts in a separate PR.

Reply:
> There must be a reason why many (all?) other `t.*` modules have this line. If this line is indeed not needed, it should be removed from all python scripts in a separate PR.

It can only be of historical reasons. A rest from the Python 2 era (and the transitional period to 3). Git history date them to some 10 years ago (~ G7.8.0). If you grep over all source code you can find this at random, but far from every .py file. I also noted the python shebang was missing (deliberately?) from some files. You are right, this is a case for a batch update all over source base.

Reply:
Addressed in #1461